src/data.py

```diff
@@ -127,6 +128,8 @@ def parse_events(event_list, eatery_id, event_date):
 
 def parse_food_stations(station_list, eatery_id):
   new_stations = []
+  if len(station_list) == 1 and len(station_list[0]['items']) == 0:  # no menu actually provided
```

Comment:
i believe the `python-ic` way to write the right boolean is `and not station_list[0]['items']`: https://stackoverflow.com/questions/53513/how-do-i-check-if-a-list-is-empty